docs/src/api.md

```diff
@@ -46,7 +46,7 @@ ismount | ismount
 isabspath | isabsolute
 splitdrive()[1] | drive
 N/A | root (property)
-split(p, "/") | segments (property)
+splitpath(p) | segments (property)
```

Comment:
Should we maybe also provide this method as an alias?

Reply:
I would like that

Reply:
Ok

Reply:
Can be a future PR